runtime/compiler/env/j9methodServer.hpp

```diff
@@ -52,6 +52,25 @@ TR_ResolvedJ9JITServerMethodInfoStruct
    void *addressContainingIsOverriddenBit;
    J9ClassLoader *classLoader;
    bool isLambdaFormGeneratedMethod;
+
+   bool operator==(const TR_ResolvedJ9JITServerMethodInfoStruct& other) const 
+      {
+      return
+            literals == other.literals &&
+            ramClass == other.ramClass &&
+            methodIndex == other.methodIndex &&
+            jniProperties == other.jniProperties &&
+            jniTargetAddress == other.jniTargetAddress &&
+            isInterpreted == other.isInterpreted &&
+            isJNINative == other.isJNINative &&
+            isMethodInValidLibrary == other.isMethodInValidLibrary &&
+            mandatoryRm == other.mandatoryRm &&
+            rm == other.rm &&
+            startAddressForJittedMethod == other.startAddressForJittedMethod &&
+            virtualMethodIsOverridden == other.virtualMethodIsOverridden &&
+            addressContainingIsOverriddenBit == other.addressContainingIsOverriddenBit &&
+            classLoader == other.classLoader;
```

Comment:
`isLambdaFormGeneratedMethod` was skipped here. Is there a reason?

Reply:
No, it should be included in the comparison.

Reply:
Note that we shouldn't be comparing `remoteMirror` attribute, because it will always be different, since we are creating a new mirror on the client with each remote request.